platform/default/headless_view.cpp

```diff
@@ -19,7 +18,7 @@
 namespace mbgl {
 
 HeadlessView::HeadlessView(float pixelRatio_, uint16_t width, uint16_t height)
-    : display(std::make_shared<HeadlessDisplay>()), pixelRatio(pixelRatio_) {
+    : display(std::move(std::make_shared<HeadlessDisplay>())), pixelRatio(pixelRatio_) {
```

Comment:
Here you don't have to `move` as `make_shared` already returns a rvalue, and thus will be moved into the `display` variable by default.